polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -86,17 +82,30 @@ bool isCandidate(LoopLikeOpInterface loop) {
     return false;
   }
 
+  WalkResult walkResult =
```

Comment:
do we care if the non-innermost loop is affine or scf loops?
thinking if we should only check the innermost loop.

Reply:
At this point we do not care. I think the loop tiling utilities would be able to tile an innermost loop contained in an affine for loop (and viceversa). I can change it.